apps/admin-x-activitypub/src/components/layout/Header.tsx

```
@@ -0,0 +1,53 @@
+import {useRouting} from '@tryghost/admin-x-framework/routing';
+
+interface HeaderProps {
+    route: string;
+}
+
+// TODO: replace with React router
+const getTitle = (route: string) => {
+    switch (route) {
+    case 'inbox':
+        return 'Inbox';
+        break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)